src/utils/yelpFusionApiUtils.ts

```diff
@@ -19,18 +19,18 @@ export class YelpFusionApiUtils {
 				latitude: geocode.lat,
 				longitude: geocode.lng,
 			})
-			.then((response) => response.jsonBody.businesses[0].id)
-			.then((id) => this.yelpClient.business(id))
-			.then((businessResult) =>
+			.then((response: any) => response.jsonBody.businesses[0].id)
+			.then((id: any) => this.yelpClient.business(id))
+			.then((businessResult: any) =>
 				businessResult.jsonBody.photos.map(
-					(photo) =>
+					(photo: any) =>
```

Comment:
I think this is a `string`? cc @DawYeong

Reply:
Yea `photo` should be a `string`.

Reply:
then a more proper name for it is probably `photoRef`